demos/image-classification/index.js

```diff
@@ -181,6 +183,18 @@ const main = async () => {
     }
   } catch (err) {
     log(`[Error] ${err}`);
+
+    status.setAttribute("class", "red");
+    info.innerHTML = `
+            ${error}<br>
+            Your device probably doesn't have a supported GPU.`;
+    label_uploadImage.setAttribute("class", "disabled");
+    uploadImage.disabled = true;
```

Comment:
Hmm, does this get re-enabled if you switch back to another device via the buttons above?

Reply:
Yes it does! Just double checked. Good catch